code-injection-analysis/abiank_redisJSON/web.js

```
@@ -0,0 +1,191 @@
+        //if (debug) console.log(sys.inspect(q));
+        for (each in q) {
+            response = response + "[" + Math.round((d - q[each]) / 1000) + "]  " + each + " : " + q[each] + "  <a href='" + whost + "/HDEL/" + req.params.hashname + "/" + each + "' target='_blank'>delete</a> <br>";
+        }
+        res.send(response);
+    });
+});
+app.post('/redispost/:key/:field', function (req, res) {
+    if (debug) console.log("posting to redis " + key + " " + field);
+    if (debug) console.log(req.body.contents);
+    // contents must be valid JSON, this has never been tested and will most likely fail
+    redis.hset(req.params.key, req.params.field, eval(req.body.contents), function (err, quotes) {
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)